explorer/src/components/account/ConfigAccountSection.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+
+import { Account, useFetchAccountInfo } from "providers/accounts";
+import { TableCardBody } from "components/common/TableCardBody";
+import { ConfigAccount } from "validators/accounts/config";
+import {
+  AccountAddressRow,
+  AccountHeader,
+  AccountOwnerRow,
+} from "components/common/Account";
+
+export function ConfigAccountSection({
+  account,
+  configAccount,
+}: {
+  account: Account;
+  configAccount: ConfigAccount;
+}) {
+  switch (configAccount.type) {
+    case "stakeConfig":
+      return (
+        <StakeConfigCard account={account} configAccount={configAccount} />
+      );
+    case "validatorInfo":
+      return (
+        <ValidatorInfoCard account={account} configAccount={configAccount} />
+      );
+  }
+}
+
+function StakeConfigCard({
+  account,
+  configAccount,
+}: {
+  account: Account;
+  configAccount: ConfigAccount;
```

Comment:
Agreed. I initially was going to union the two config account info types under ConfigAccount (I think this would cause TS2339 property doesn't exist from the or'ing two types with unique members), but I think I should break them out into ValidatorInfoAccount and StakeConfigAccount. Does that sound right to you?

Reply:
Typically I try to follow this pattern: https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions Do you think that would help with the missing property error?

Reply:
I think so!